plugins-bundled/internal/plugin-catalog-app/src/components/PluginList.tsx

```diff
@@ -23,7 +23,7 @@ export const PluginList = ({ plugins }: Props) => {
         return (
           <Card
             key={`${orgName}-${name}-${typeCode}`}
-            href={`${PLUGIN_ROOT}?tab=plugin&slug=${slug}`}
+            href={`${PLUGIN_ROOT}/plugin/${slug}`}
```

Comment:
Wouldn't it be nice to have a function as part of the API that gives you the proper path? I'm guessing you have a context that the app can operate within?